tests/e2e-openshift/gateway/00-install.yaml

```diff
@@ -1,3 +1,8 @@
+# The namespace is auto-deleted by kuttl after the test run.
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: kuttl-ocp-gateway
```

Comment:
Is this change needed? I think it would be enough setting the `namespace` field in `kuttl-test-openshift.yaml` (as you did). From the [KUTTL documentation](https://kuttl.dev/docs/testing/reference.html#testsuite):
> The namespace to use for tests. This namespace will be created if it does not exist and removed if it was created (unless skipDelete is set). If no namespace is set, one will be auto-generated.

Reply:
it was not created when I run it locally